src/test/groovy/com/github/vkorobkov/jfixtures/loader/RowTest.groovy

```diff
@@ -94,4 +94,70 @@ class RowTest extends Specification {
         expect:
         EqualsVerifier.forClass(Row).verify()
     }
+
+    def "#columns(Object...) returns row from key/value pairs"() {
+        given:
+        def row = new Row("vlad", columns)
+
+        when:
+        def result = row.columns(
+                "name", "Vlad",
+                "age", 30,
+                "hobby", "sleep"
+        )
+
+        then:
+        with(result.columns){
+            size() == 4
+            toMapString() == "[id:Value(value=1, type=AUTO), name:Value(value=Vlad, type=TEXT), age:Value(value=30, type=AUTO), hobby:Value(value=sleep, type=TEXT)]"
+        }
+    }
+
+    def "#columns(Object...) throws IllegalArgumentException when odd number of key/value pairs"() {
+        given:
+        def row = new Row("vlad", columns)
+
+        when:
+        row.columns("name", "Vlad", "age")
+
+        then:
+        def exception = thrown(IllegalArgumentException)
+        exception.message == "Parameter <keyValuePairs> is expected to have odd length since it represents key/value pairs"
+    }
+
+    def "#columns(Object...) throws IllegalArgumentException when odd object is not a string"() {
+        given:
+        def row = new Row("vlad", columns)
+
+        when:
+        row.columns(1, "Vlad", "age", 30)
+
+        then:
+        def exception = thrown(IllegalArgumentException)
+        exception.message == "Column name is expected to be a string, but was passed class = [class java.lang.Integer], value = [1]"
+    }
+
+    def "#columns(Object...) throws IllegalArgumentException when an empty array passed"() {
+        given:
+        def row = new Row("vlad", columns)
+
+        when:
+        row.columns([])
+
+        then:
+        def exception = thrown(IllegalArgumentException)
```

Comment:
Why the exception is thrown in this case? We probably should allow empty array

Reply:
To me "no new columns" is a valid case and user does not ever has to write a code like:
```java
Object[] myNewColumns = ....
if (myNewColumns.length == 0) {
  return row;
else {
  return row.columns(myNewColumns);
}
```

Reply:
Is it correct to pass null? like row.columns(null)

Reply:
Nope, `null` is unexpected sine the argument is not `Optional`

Reply:
Btw I explained the thing badly. Null is never expected as an argument, never. If argument needs to be nullable, class `Optional` should be used.

Empty collection is a valid collection and normally should not throw errors if passed as an arg - it should just give no effect.

Reply:
Should we handle this case somehow when null passed? we will get NPE

Reply:
NPE is good enough, just let it fail